src/scss/components/_media_box.scss

```diff
@@ -15,35 +15,38 @@
   color: get-color((color: blackNeutral, opacity: .6));
 
   &__title {
-    color: get-color(punch);
+
+    .caption-title--mini {
```

Comment:
My point about this selector: https://github.com/getninjas/gaiden/pull/99/files#diff-08fca2c2524b8f14547a7595bce358f7R154